chroma-manager/Makefile

```diff
@@ -208,6 +208,13 @@ rpms: cleandist tarball
 	rm -rf _topdir
 	echo "jenkins_fold:end:Make Manager RPMS"
 
+install_requirements: requirements
+	echo "jenkins_fold:start:Install Python requirements"
+	pip install --upgrade pip;                              \
+	pip install --upgrade setuptools;                       \
+	pip install -r requirements.txt
+	echo "jenkins_fold:end:Install Python requirements"
+
```

Comment:
[Here's a technique](http://blog.bottlepy.org/2012/07/16/virtualenv-and-makefiles.html) to avoid doing all of that virtualenv configuration (i.e. every single time even) if the `requirements.txt` file has not actually changed.